integration/src/test/scala/com/snowplowanalytics/snowplow/collectors/scalastream/intergation/TestUtils.scala

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0, and
+ * you may not use this file except in compliance with the Apache License
+ * Version 2.0.  You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the Apache License Version 2.0 is distributed on an "AS
+ * IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.collectors.scalastream.intergation
+
+import cats.effect.{ContextShift, Resource, Sync, Timer}
+import cats.implicits._
+import com.amazonaws.auth.{AWSStaticCredentialsProvider, BasicAWSCredentials}
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration
+import com.amazonaws.services.kinesis.model.GetRecordsRequest
+import com.amazonaws.services.kinesis.{AmazonKinesis, AmazonKinesisClientBuilder}
+import com.snowplowanalytics.snowplow.eventgen.collector.Api
+import com.snowplowanalytics.snowplow.eventgen.tracker.{HttpRequest => RequestStub}
+import com.snowplowanalytics.snowplow.eventgen.tracker.HttpRequest.Method.{Get, Head, Post}
+import org.apache.commons.codec.binary.{Base64 => ABase64}
+import org.scalacheck.Gen
+
+import java.net.URI
+import java.net.http.HttpRequest.BodyPublishers
+import java.net.http.HttpResponse.BodyHandlers
+import java.net.http.{HttpClient, HttpRequest, HttpResponse}
+import java.nio.charset.StandardCharsets
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import scala.concurrent.duration.DurationInt
+import scala.jdk.CollectionConverters.iterableAsScalaIterableConverter
+
+object TestUtils {
+  object Kinesis {
```

Comment:
Would be nice to place this object into its own `Kinesis.scala`. Same for `Http`. We can create a new package `utils`.

Reply:
Done.